recipe/meta.yaml

```diff
@@ -59,7 +60,6 @@ test:
     - check-for-warnings.py
     - allowed-warnings-base.txt
   commands:
-    - pytensor-cache help
```

Comment:
We should keep this.

Reply:
Woops! I thought I added it back in after changing the entry point. Thanks for the catch!